styles/codeberg/catppuccin.user.css

```diff
@@ -105,14 +105,22 @@
     }
   }
 
+  [data-theme="codeberg-light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  
+  [data-theme="codeberg-dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
   @media (prefers-color-scheme: light) {
-    :root {
+    [data-theme="codeberg-auto"] {
       #catppuccin(@lightFlavor, @accentColor);
     }
   }
 
   @media (prefers-color-scheme: dark) {
-    :root {
+    [data-theme="codeberg-auto"] {
       #catppuccin(@darkFlavor, @accentColor);
     }
```

Comment:
This won't work with our selectors above for Gitea in general? Above it just does prefers-color-scheme... so if you prefer dark but have Codeberg set to light you'll have a mismatch.

Reply:
thats fair